src/main/java/stsjorbsmod/actions/CullCardAction.java

```diff
@@ -0,0 +1,62 @@
+package stsjorbsmod.actions;
+
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.helpers.GetAllInBattleInstances;
+import com.megacrit.cardcrawl.vfx.combat.FlashAtkImgEffect;
+
+import java.util.Iterator;
+import java.util.UUID;
+
+public class CullCardAction extends AbstractGameAction {
```

Comment:
Did you mean to include this? It doesn't look like CULLCard is using it anymore

Reply:
It will be needed to upgrade on first swing at the end of combat, and upgrade only once if combat not finished.